docs/QuickStart.md

```diff
@@ -0,0 +1,201 @@
+
+
+## SQL Setup
+
+Next you'll configure your SQL Server database for use with Azure SQL binding for Azure Functions.
+
+This will require connecting to and running queries - you can use [Azure Data Studio](https://docs.microsoft.com/sql/azure-data-studio/download-azure-data-studio) or the [MSSQL for VS Code Extension](https://docs.microsoft.com/sql/tools/visual-studio-code/sql-server-develop-use-vscode) to do this.
+
+1. First you'll need a table to run queries against. If you already have one you'd like to use then you can skip this step.
+
+    Otherwise connect to your database and run the following query to create a simple table to start with.
+
+```sql
+CREATE TABLE Employees (
+        EmployeeId int,
+        FirstName varchar(255),
+        LastName varchar(255),
+        Company varchar(255),
+        Team varchar(255)
+);
+```
+
+2. Next a primary key must be set in your SQL table before using the bindings. To do this, run the queries below, replacing the placeholder values for your table and column.
+
+```sql
+ALTER TABLE ['{table_name}'] ALTER COLUMN ['{primary_key_column_name}'] int NOT NULL
+
+ALTER TABLE ['{table_name}'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['{primary_key_column_name}']);
+```
+
+
+## Create a Function App
+
+Now you will need a Function App to add the binding to. If you have one created already you can skip this step.
+
+These steps can be done in the Terminal/CLI or with PowerShell.
+
+1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
+
+2. Create a function app for .NET, JavaScript, TypeScript, Python or Java.
+
+    **.NET**
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime dotnet
+    ```
+
+    **JavaScript (NodeJS)**
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime node --language javascript
+    ```
+
+    **TypeScript (NodeJS)**
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime node --language typescript
+    ```
+
+    **Python**
+
+    *See [#250](https://github.com/Azure/azure-functions-sql-extension/issues/250) before starting.*
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime python
+    ```
+
+    **Java**
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime java
+    ```
+
+3. Enable SQL bindings on the function app. More information can be found [in Microsoft Docs](https://docs.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
+
+    **.NET:** Install the extension.
+
+    ```powershell
+    dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --prerelease
+    ```
+
+    **JavaScript and TypeScript:** Update the `host.json` file to the preview extension bundle.
+    ```json
+    "extensionBundle": {
+        "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+        "version": "[4.*, 5.0.0)"
```

Comment:
Shouldn't this be included in each of the language's Binding Guides? 
Users can now directly click on the language details and skip over this part.